certora/specs/compoundMath.spec

```diff
@@ -1,34 +1,35 @@
 methods {
-    mul(uint256, uint256)   returns (uint256)   envfree
-    div(uint256, uint256)   returns (uint256)   envfree
+    function mul(uint256, uint256)   external returns (uint256)   envfree;
+    function div(uint256, uint256)   external returns (uint256)   envfree;
 }
 
 definition WAD()  returns uint = 10^18;
+definition UINT_LIMIT() returns mathint = 2 ^ 255 * 2;
```

Comment:
what about
```suggestion
definition MAX() returns mathint = 2 ^ 255 * 2;
```

Reply:
Why can't we `2 ^ 256`? 😆

Reply:
Sorryy

Reply:
I find `MAX` to be a bit misleading because the max of a set should be in that set

Reply:
ah yes indeed. We could use 2**256 - 1 but that may not be practical for tests after

Reply:
We will switch back to the previous notation anyway once the issue is resolved on the Certora side